docs/ReadmeExample.md

```diff
@@ -59,6 +61,9 @@ import java.util.concurrent.Executors
 import scala.concurrent.ExecutionContext
 
 implicit val cs: ContextShift[IO] = IO.contextShift(scala.concurrent.ExecutionContext.Implicits.global)
+
+//note: this should be shut down when it's no longer necessary - normally that's at the end of your app.
+//See the whole README example for proper resource management in terms of ExecutionContexts.
 val blockingExecutionContext = ExecutionContext.fromExecutorService(Executors.newFixedThreadPool(2))
```

Comment:
This is a little confusing to me in that the changes above name `blockingExecutionContext: Resource[F, EC]` but this is `blockingExecutionContext: EC`.

I'm concerned that a reader unfamiliar with `Resource` won't quite understand the implications and end up with opening a resource multiple times instead of opening it once and then sharing it.

Perhaps we might name the `Resource` versions with `make` or `get` or something like that in the name?